src/platform/intel/cavs/pm_runtime.c

```diff
@@ -164,9 +163,15 @@ static inline void cavs_pm_runtime_dis_dwdma_clk_gating(uint32_t index)
 	shim_write(SHIM_CLKCTL, shim_read(SHIM_CLKCTL) |
 		   SHIM_CLKCTL_LPGPDMAFDCGB(index));
 
-	trace_event(TRACE_CLASS_POWER,
-		    "dis-dwdma-clk-gating index %d CLKCTL %08x",
+	trace_power("dis-dwdma-clk-gating index %d CLKCTL %08x",
 		    index, shim_read(SHIM_CLKCTL));
+#elif defined(CONFIG_CANNONLAKE)
```

Comment:
BUG.
you cannot have an if defined (CNL) ... elif defined (CNL)... The latter will never be executed.

Reply:
I'm pretty sure You've got mislead by github which collapsed the source diff. It's #ifdef(APL) #elif def(CNL)